README.md

```diff
@@ -1 +1,17 @@
-# -Hello-Microverse-
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] [OPTIONAL] Kindly consider removing `test.md` file from your repo, this is a requirement but the first reviewer forget to mention it